keyboards/alpaca/wfeclipse/info.json

```diff
@@ -0,0 +1,210 @@
+            {"matrix": [2, 3], "x": 51, "y": 30, "flags": 1},
+            {"matrix": [2, 2], "x": 39, "y": 30, "flags": 1},
+            {"matrix": [2, 1], "x": 27, "y": 30, "flags": 1},
+            {"matrix": [2, 0], "x": 6, "y": 30, "flags": 1},
+            {"matrix": [1, 15], "x": 198, "y": 18, "flags": 1},
+            {"matrix": [1, 14], "x": 186, "y": 18, "flags": 1},
+            {"matrix": [1, 13], "x": 168, "y": 18, "flags": 1},
+            {"matrix": [1, 12], "x": 156, "y": 18, "flags": 1},
+            {"matrix": [1, 11], "x": 144, "y": 18, "flags": 1},
+            {"matrix": [1, 10], "x": 132, "y": 18, "flags": 1},
+            {"matrix": [1, 9], "x": 120, "y": 18, "flags": 1},
+            {"matrix": [1, 8], "x": 108, "y": 18, "flags": 1},
+            {"matrix": [1, 7], "x": 96, "y": 18, "flags": 1},
+            {"matrix": [1, 6], "x": 84, "y": 18, "flags": 1},
+            {"matrix": [1, 5], "x": 72, "y": 18, "flags": 1},
+            {"matrix": [1, 4], "x": 60, "y": 18, "flags": 1},
+            {"matrix": [1, 3], "x": 48, "y": 18, "flags": 1},
+            {"matrix": [1, 2], "x": 36, "y": 18, "flags": 1},
+            {"matrix": [1, 1], "x": 24, "y": 18, "flags": 1},
+            {"matrix": [1, 0], "x": 6, "y": 18, "flags": 1},
+            {"matrix": [0, 15], "x": 198, "y": 6, "flags": 1},
+            {"matrix": [0, 14], "x": 186, "y": 6, "flags": 1},
+            {"matrix": [0, 13], "x": 162, "y": 6, "flags": 1},
+            {"matrix": [0, 12], "x": 150, "y": 6, "flags": 1},
+            {"matrix": [0, 11], "x": 138, "y": 6, "flags": 1},
+            {"matrix": [0, 10], "x": 126, "y": 6, "flags": 1},
+            {"matrix": [0, 9], "x": 114, "y": 6, "flags": 1},
+            {"matrix": [0, 8], "x": 102, "y": 6, "flags": 1},
+            {"matrix": [0, 7], "x": 90, "y": 6, "flags": 1},
+            {"matrix": [0, 6], "x": 78, "y": 6, "flags": 1},
+            {"matrix": [0, 5], "x": 66, "y": 6, "flags": 1},
+            {"matrix": [0, 4], "x": 54, "y": 6, "flags": 1},
+            {"matrix": [0, 3], "x": 42, "y": 6, "flags": 1},
+            {"matrix": [0, 2], "x": 30, "y": 6, "flags": 1},
+            {"matrix": [0, 1], "x": 18, "y": 6, "flags": 1},
+            {"matrix": [0, 0], "x": 6, "y": 6, "flags": 1},
+            {"x": 125, "y": 1, "flags": 4},
+            {"x": 125, "y": 2, "flags": 4},
+            {"x": 125, "y": 3, "flags": 4},
+            {"x": 125, "y": 4, "flags": 4},
+            {"x": 125, "y": 5, "flags": 4},
+            {"x": 125, "y": 6, "flags": 4},
+            {"x": 125, "y": 7, "flags": 4},
+            {"x": 125, "y": 8, "flags": 4},
+            {"x": 125, "y": 9, "flags": 4},
+            {"x": 125, "y": 10, "flags": 4},
+            {"x": 125, "y": 11, "flags": 4},
+            {"x": 125, "y": 12, "flags": 4}
+        ],
+        "max_brightness": 50
+    },
+    "usb": {
+        "device_version": "1.0.4",
+        "pid": "0x0038",
+        "vid": "0x308F"
```

Comment:
Sold by/at kono.store (input club).

Reply:
Sure. This keyboard is design for input club and we can provide the Vendor ID registration form and payment invoice for this VID:0x308F if necessary. Thanx!